pkg/sharding_engine/simple/sql_se.go

```diff
@@ -0,0 +1,75 @@
+package simple
+
+import (
+	"context"
+	"fmt"
+	pb "github.com/anujga/dstk/pkg/api/proto"
+	"github.com/jmoiron/sqlx"
+	"go.uber.org/zap"
+	"strings"
+	"time"
+)
+
+type sqlSe struct {
+	db *sqlx.DB
+}
+
+func (s *sqlSe) CreatePartition(ctx context.Context, request *pb.PartitionCreateRequest) (*pb.PartitionRpcBaseResponse, error) {
```

Comment:
Add a comment for semantics. When all does the api fail for non system issues. eg:
fails if id already exists.

Reply:
will add all the details when we merge update and create with enums for various kinds of updations.